__tests__/api.test.js

```diff
@@ -170,4 +170,71 @@ describe('/api/articles/:article_id/comments', () => {
             })
         })
     })
+    describe('POST', () => {
+        it('targets an article by its id and adds a comment to it', () => {
+            const comment = {
+                username: 'butter_bridge',
+                body: 'This is my random comment'
+            }
+            return request(app)
+            .post('/api/articles/7/comments')
+            .send(comment)
+            .expect(201)
+            .then(({body: {commentPosted}}) => {
+                expect(commentPosted).toBe('This is my random comment')
+            })
```

Comment:
We could test what happens when everything is correct but extra properties are sent, which should be ignored :)